auth_mellon_httpclient.c

```diff
@@ -1,7 +1,7 @@
 /*
  *
  *   mod_auth_mellon.c: an authentication apache module
- *   Copyright � 2003-2007 UNINETT (http://www.uninett.no/)
+ *   Copyright � 2003-2007 UNINETT (http://www.uninett.no/)
```

Comment:
There seem to be a character encoding issue here, are you perhaps using non-utf8 while creating pacthes ?

Reply:
This is wired. I am in UTF-8 and have not change this line.